azerothcore/azerothcore-wotlk
# fix(DB/Gameobject): removes non-armor drops from armor crate

## Changes Proposed:
-  Removes non-armor drops from armor crate 

## Issues Addressed:

- Closes #5724
- Closes https://github.com/chromiecraft/chromiecraft/issues/567

## SOURCE:

Both [Wowhead](https://tbc.wowhead.com/object=3703/armor-crate#contains) and [Gamepedia](https://wowpedia.fandom.com/wiki/Armor_Crate?oldid=2491611) indicates only grey armor drops, moreover the name states the same.

## Tests Performed:

- As below.


## How to Test the Changes:


1. Run SQL query - Should change 69 rows
2. Check that entry 3703 only drops armour of white/grey type
    - In Keira — Gameobject - 3703 => Gameobject Loot
    - Or with query — ``SELECT * FROM `gameobject_loot_template` WHERE `entry` = 1619;``
3. Check in game — ``.go o 32254``

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
